pom.xml

```diff
@@ -1550,20 +1564,27 @@
 
         <profile>
             <id>spark-3.0</id>
+            <properties>
+                <spark.version>3.0.2</spark.version>
+                <delta.version>0.8.0</delta.version>
+                <maven.plugin.scalatest.exclude.tags></maven.plugin.scalatest.exclude.tags>
+            </properties>
         </profile>
 
         <profile>
-            <id>spark-3.2-snapshot</id>
+            <id>spark-3.1</id>
             <properties>
-                <spark.version>3.2.0-SNAPSHOT</spark.version>
+                <spark.version>3.1.1</spark.version>
+                <delta.version>1.0.0</delta.version>
+                <maven.plugin.scalatest.exclude.tags>org.apache.kyuubi.tags.ExtendedSQLTest,org.apache.kyuubi.tags.IcebergTest</maven.plugin.scalatest.exclude.tags>
             </properties>
         </profile>
 
         <profile>
-            <id>spark-3.1</id>
+            <id>spark-3.2-snapshot</id>
```

Comment:
spark-master？

Reply:
just reordered `spark-3.1` and `spark-3.2-snapshot`

Reply:
sounds good